docs/Nodes.md

```
@@ -4,7 +4,7 @@
 
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**_Nodes** | [**List&lt;Node&gt;**](Node.md) |  | 
+**VarNodes** | [**List&lt;Node&gt;**](Node.md) |  | 
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)